api/src/main/java/com/capitalone/dashboard/request/PullRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.capitalone.dashboard.request;
+
+import org.bson.types.ObjectId;
+
+import javax.validation.constraints.NotNull;
+
+public class PullRequest {
+    @NotNull
+    private ObjectId componentId;
+    private Integer numberOfDays;
+
+    public ObjectId getComponentId() {
+        return componentId;
+    }
+
+    public void setComponentId(ObjectId componentId) {
+        this.componentId = componentId;
+    }
+
+    public Integer getNumberOfDays() {
+        return numberOfDays;
+    }
+
+    public void setNumberOfDays(Integer numberOfDays) {
+        this.numberOfDays = numberOfDays;
+    }
+
+}
```

Comment:
Pull and Issue differ from one field - the merge date. In future, if we were to do graphs based on Pulls merge and close dates, it might be a good idea to keep them distinct. GIT treats them as distinct so I carried the same thought process here. Let me know if I should merge both into one class please